pkg/libvirttools/virtualization.go

```diff
@@ -167,7 +167,6 @@ func (v *VirtualizationTool) processVolumes(containerName string, mounts []*kube
 				return copyDomXML, err
 			}
 			domXML = string(outArr[:])
-			glog.Infof("Creating domain:\n%s", domXML)
```

Comment:
Please don't remove this one, it's very useful for troubleshooting, it prints the overal vm xml definition after adding volumes, if there were specified for pod.